docs/user/filter_expressions.rst

```diff
@@ -0,0 +1,205 @@
+
+JEXL also supports lists and objects (known as dictionaries in other languages)
+as well as attribute access:
+
+.. code-block:: javascript
+
+   [1, 2, 1].length // == 3
+   {foo: 1, bar: 2}.foo // == 1
+
+Unlike JavaScript, JEXL supports an ``in`` operator for checking if a substring
+is in a string or if an element is in an array:
+
+.. code-block:: javascript
+
+   "bar" in "foobarbaz" // == true
+   3 in [1, 2, 3, 4] // == true
+
+The context passed to JEXL can be expressed using identifiers, which also
+support attribute access:
+
+.. code-block:: javascript
+
+   normandy.locale == 'en-US' // == true if the client's locale is en-US
+
+Another unique feature of JEXL is transforms, which modify the value given to
+them. Transforms are applied to a value using the ``|`` operator, and may take
+additional arguments passed in the expression:
+
+.. code-block:: javascript
+
+   'January 7th, 1980'|date // == a date object
+
+.. _JEXL Readme: https://github.com/TechnologyAdvice/Jexl#jexl---
+
+Context
+-------
+This section defines the context passed to filter expressions when they are
+evaluated. In other words, this is the client information available within
+filter expressions.
+
+.. js:data:: normandy
+
+   The ``normandy`` object contains general information about the client.
+
+.. js:attribute:: normandy.version
+
+   **Example:** ``'47.0.1'``
+
+   String containing the user's Firefox version.
+
+.. js:attribute:: normandy.channel
+
+   String containing the update channel. Valid values include, but are not
+   limited to:
+
+   * ``'release'``
+   * ``'aurora'``
+   * ``'beta'``
+   * ``'nightly'``
+   * ``'default'`` (self-built or automated testing builds)
+
+.. js:attribute:: normandy.isDefaultBrowser
+
+   Boolean specifying whether Firefox is set as the user's default browser.
+
+.. js:attribute:: normandy.searchEngine
+
+   **Example:** ``'google'``
+
+   String containing the user's default search engine identifier.
+
+.. js:attribute:: normandy.syncSetup
+
+   Boolean containing whether the user has set up Firefox Sync.
+
+.. js:attribute:: normandy.plugins
+
+   An object mapping of plugin names to :js:class:`Plugin` objects describing
+   the plugins installed on the client.
+
+.. js:attribute:: normandy.locale
+
+   **Example:** ``'en-US'``
+
+   String containing the user's locale.
+
+.. js:attribute:: normandy.country
+
+   **Example:** ``'US'``
+
+   `ISO 3166-1 alpha-2`_ country code for the country that the user is located
+   in. This is determined via IP-based geolocation.
+
+   .. _ISO 3166-1 alpha-2: https://en.wikipedia.org/wiki/ISO_3166-1
+
+.. js:attribute:: normandy.request_time
+
+   Date object set to the time and date that the user requested recipes from
+   Normandy. Useful for comparing against date ranges that a recipe is valid
+   for.
+
+   .. code-block:: javascript
+
+      // Do not run recipe after January 1st.
+      normandy.request_time < '2011-01-01'|date
+
+Transforms
+----------
```

Comment:
Current master doesn't have `stableSample` available yet so I didn't include it.